keel/api/v1/document/views.py

```diff
@@ -0,0 +1,46 @@
+from django.http import FileResponse
+
+from rest_framework.viewsets import GenericViewSet
+from rest_framework.permissions import IsAuthenticated, AllowAny
+from rest_framework import mixins, viewsets, status
+from rest_framework.response import Response
+
+from keel.document.models import Documents
+
+from .serializers import DocumentsSerializer
+
+
+
+class GetDocument(GenericViewSet):
+
+    def generate(self, request, format = 'json', **kwargs):
+        
+        response = {
+                "status": 0,
+                "message":"Document Fetched successfully",
+                "data": ""
+        }
+
+        doc_id = kwargs.get('doc_id')
+
+        if not doc_id:
+        	response['status'] = 1
+        	response['message'] = "Document Id is invalid"
```

Comment:
this is not a invalid Document scenario...but instead an invalid request!!

ideally throw a 400 here